app/assets/javascripts/components/miq-button.js

```diff
@@ -14,6 +15,7 @@ ManageIQ.angular.app.component('miqButton', {
         this.onClick();
       }
       event.preventDefault();
+      event.target.blur();
```

Comment:
It's to address a lingering focus on the button when it's clicked.
Very much pronounced when you click the Reset button, since for Reset we do not redirect to the `show_list` page.